### DISCUSSION TITLE
I am a verified Teacher, I cannot access Copilot. is it normal ?

### DISCUSSION THREAD

**Discussion by mukhtarhussainbaloch** [community/community#44987] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am a verified Teacher, I cannot access Copilot. is it normal? When I try to access the service through IntelliJ or VS Code plugin  I get the Error "No access to GitHub Copilot Found".

**Comment by davecheney** [comment]
@mukhtarhussainbaloch im sorry to hear you are experiencing difficulties. Do you have a an active faculty coupon on your GitHub account?

**Reply by mukhtarhussainbaloch** [reply]
Thank you for the quick response. Where can I check it? 
on https://education.github.com/discount_requests/pack_application page on I have "Hi, Mukhtar Hussain! You were last verified as a teacher on Jan 18, 2023. It is not necessary for you to re verify at this time."

**Reply by davecheney** [reply]
You will have received a coupon code, that coupon needs to be active on your GitHub account to qualify for complementary copilot access

**Reply by mukhtarhussainbaloch** [reply]
I haven't received any coupons from GitHub via email. What should I do now ?

**Comment by davecheney** [comment]
Thank you, I have confirmed your coupon is active, however you didn't complete the copilot setup steps. Please go to https://github.com/settings/copilot and complete the required steps, then 

<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by mukhtarhussainbaloch** [reply]
In my account https://github.com/settings/copilot looks like this 
![github-copilot](https://user-images.githubusercontent.com/50365511/213906889-62db0c2e-9ff5-402e-aa7d-ba4085510448.png)
should I apply for the 60 days trail?

**Reply by davecheney** [reply]
Please contact support, https://support.github.com/contact

**Comment by iamcjwilliams** [comment]
@mukhtarhussainbaloch Which account or email did use to verify your teacher status? and has a GitHub account been created for the email that was used to verify?

**Reply by mukhtarhussainbaloch** [reply]
My issue is resolved by GitHub Support. Thank you
